comptests/s3/test_rest_api_rgw.py

```diff
@@ -269,16 +259,331 @@ def test_user_create_36665(self):
+        status, user_info = loop.run_until_complete(self.obj.create_user(user_params1))
+        self.log.info(user_info)
+        self.log.info("Step 3: Verifying that new IAM user is created successfully")
+        assert status == HTTPStatus.OK, "Not able to create user. Test Failed"
+        self.log.info("Step 4: IAM user is created successfully")
+        policy = {
+            "Version": "2012-10-17",
+            "Statement": [
+                {
+                    "Effect": "Deny",
+                    "Action": "s3:PutObject",
+                    "Resource": "arn:aws:s3:::test1/*"
+                }
+            ]
+        }
+        self.log.info("Step 5: Applying user policy to the user")
+        self.log.info(policy)
+        policy_document = json.dumps(policy)
+        user_params2 = {
+            'UserName' : user_name,
+            'PolicyName' : 'Policy1',
+            'PolicyDocument' : policy_document,
+            'Action' : 'PutUserPolicy',
+            'format' : 'json'
+        }
+        loop = asyncio.get_event_loop()
+        status = loop.run_until_complete(self.obj.put_user_policy(user_params2))
+        self.log.info("Step 6: verifying Applied user policy to the user")
+        assert status[0] == HTTPStatus.OK, "Not able to apply policy"
+        self.log.info("Step 7:  Applied user policy Successfully")
+        user_params3 = {
+            'UserName' : user_name,
+            'PolicyName' : 'Policy1',
+            'Action' : 'DeleteUserPolicy',
+            'format' : 'json'
+        }
+        loop = asyncio.get_event_loop()
+        status = loop.run_until_complete(self.obj.delete_user_policy(user_params3))
+        assert status[0] == HTTPStatus.OK, "Not able to delete policy"
+        self.log.info("Step 8: IAM policy deleted successfully")
+        self.log.info("Step 9: Deleting created IAM user")
+        self.created_users.append(user_params1)
+        self.log.info("END: %s",test_case_name)
+
+    @pytest.mark.api_user_ops
+    @pytest.mark.tags('TEST-41408')
+    def test_41408(self):
+        """ Test to DeleteUserPolicy to the InValid user. """
+        test_case_name = cortxlogging.get_frame()
+        self.log.info("##### Test started -  %s #####", test_case_name)
+        self.log.info("Test to DeleteUserPolicy for invalid user.")
+        user_name = f"{self.user_name_prefix}{str(time.perf_counter_ns()).replace('.', '_')}"
+        policy_document = {
+            "Version": "2012-10-17",
+            "Statement": [
+                {
+                    "Effect": "Deny",
+                    "Action": "s3:PutObject",
+                    "Resource": "arn:aws:s3:::test1/*"
+                }
+            ]
+        }
+        self.log.info("Step 1: Applying user policy to invalid user")
+        self.log.info(policy_document)
+        user_params2 = {
+            'UserName' : user_name,
+            'PolicyName' : 'Policy1',
+            'PolicyDocument' : policy_document,
```

Comment:
No need to give policy_document to delete_user_policy API
ref: [IAM API Specification](https://seagate-systems.atlassian.net/wiki/spaces/PUB/pages/945619613/IAM+API+Specification)